docs/guides/attributes.adoc

```diff
@@ -0,0 +1,14 @@
+:project_name: Keycloak
+:archivebasename: keycloak client
+:archivebasename: keycloak client
+:project_doc_base_url: https://www.keycloak.org/docs/latest
+:apidocs_name: API Documentation
+:apidocs_link: https://www.keycloak.org/docs/latest/api_documentation/
+:authorizationguide_name: Authorization Services Guide
+:authorizationguide_name_short: Authorization Services
+:authorizationguide_link: {project_doc_base_url}/authorization_services/
+:kc_base_path:
```

Comment:
```suggestion
```

Could probably remove `kc_base_path` and just hard-code to `/' in the guides. Seems a bit pointless to have it here.